packages/web-app-files/src/App.vue

```diff
@@ -1,60 +1,26 @@
 <template>
   <main id="files" class="oc-flex oc-height-1-1">
     <div v-if="dragareaEnabled" class="dragarea" />
-    <div ref="filesListWrapper" tabindex="-1" class="files-list-wrapper oc-width-expand">
```

Comment:
fyi: the wrapper div has been moved into a separate component, which is used inside all relevant views, because otherwise the right sidebar would have been a child of this file list wrapper here.